src/components/dataContainer.js

```diff
@@ -81,7 +81,20 @@
         if (authProfile) {
           return (
             <B.GetMe authenticationProfileId={authProfile}>
-              {canvasLayout()}
+              {({ loading, error, data }) => {
+                if (loading) {
+                  B.triggerEvent('onUserLoading');
```

Comment:
We would like to keep some consistency in the event names. Could you change these to: `onUserLoad`, `onUserError` and `onUserSuccess`?

Reply:
Yeah good idea, done